portality/scripts/orphaned_datasets.py

```diff
@@ -0,0 +1,57 @@
+""" Check whether the DOAJ contains records with invalid country, currency or language datasets """
```

Comment:
Have you run this yet on a test dataset?  It would be interesting to see the result.